src/customer/customer-action-creator.ts

```diff
@@ -54,6 +54,24 @@ export default class CustomerActionCreator {
         };
     }
 
+    createAddress(
+        customerAddress: CustomerAddressRequestBody,
+        options?: RequestOptions
+    ): ThunkAction<CreateCustomerAddressAction, InternalCheckoutSelectors> {
+        return store => concat(
+            of(createAction(CustomerActionType.CreateCustomerAddressRequested)),
+            from(this._customerRequestSender.createAddress(customerAddress, options))
+                .pipe(
+                    switchMap(() => concat(
+                        this._checkoutActionCreator.loadCurrentCheckout(options)(store),
```

Comment:
I wonder if the customer address endpoint itself can return all the newly created address, we can then map the response in the same shape as the checkout's customer sub resource and store it in memory. That way we don't have to make an extra call to hydrate the cart, which can sometimes be slow depending on its size.

Reply:
Yeah, that would be great, or have a customer endpoint where we can retrieve addresses, avoiding having to reload the cart/checkout.

Reply:
cc @snaderiBC

Reply:
>That way we don't have to make an extra call to hydrate the cart

That's interesting idea 🤔